README.md

```diff
@@ -256,20 +256,24 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // times out, requiring another attempt at authentication.
                 "state_timeout_in_seconds": 600,
 
-                // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "irods_user_claim": "irods_username",
-
-                // The mapping of a user to the provided values. All values must
-                // be matched to map an OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "user_attribute_mapping": {
-                    "irods_username": {
-                        "sub": "123-abc-456-xyz",
-                        "email": "rods_user@example.org"
+                // User mapping plugins replace 'user_attribute_mapping' and
+                // 'irods_user_claim'. Addtional plugins may be developed to
+                // suit specific needs.
+                "user_mapping": {
+                    // Full path to plugin to load.
+                    // Currently we supply the following plugins:
+                    //   - local_file
+                    //   - user_claim
+                    "plugin_path": "/usr/lib/irods_http_api/plugins/user_mapping/libirods_http_plugin-local_file.so",
```

Comment:
> I think this needs to be something like `/path/to/user_mapping_plugin.so` since we can't define any configuration for the plugin.

I'm fine with changing the path to an example path, I don't follow the reasoning though.

> We can't point to a default file because the HTTP API doesn't ship any user mapping files.

I don't understand what is being stated here. The path that we are providing in `plugin_path` is one of the plugins that we will be shipping with the HTTP API going forth.  A default install should have this file at the path specified.

You're right that we cannot provide a 'default mapping', but this isn't the mapping. This is the path to the shared library/plugin.

Reply:
Also relevant: https://github.com/irods/irods_client_http_api/pull/331#discussion_r1759462149

Reply:
Pointing to a real plugin isn't the important bit IMO. It's specifically about informing the reader/admin of what should go there. Hence my generic suggestion.

> I don't understand what is being stated here. The path that we are providing in plugin_path is one of the plugins that we will be shipping with the HTTP API going forth. A default install should have this file at the path specified.

I see how my comment is a little confusing. My comment is also related to the removal of the `file_path` property under `configuration`. By removing the `/configuration/file_path` property, the need to specify a real user mapping plugin becomes pointless.

Reply:
Set to example path.